### PR TITLE
updated core to 1.59.2

### PR DESCRIPTION
**Pull request by h4ck3rk3y**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # TBD
 
+### Changes
+* Upgraded core to 1.59.2
```
Comment (leoporoli) [inline docs/changelog.md:4]: Will you need to upgrade the Engine also?

Comment (h4ck3rk3y) [reply docs/changelog.md:4]: i ran the script but i don't see any dependence on container-engine-lib in the go.mod or package.json

Comment (h4ck3rk3y) [reply docs/changelog.md:4]: image

no dependency on the engine server either ithink. let me see

Comment (h4ck3rk3y) [reply docs/changelog.md:4]: image

Comment (leoporoli) [reply docs/changelog.md:4]: sorry, I mean the Engine server's SDK, you've just released a new Engine server. Will you need to add this also here?

Comment (h4ck3rk3y) [reply docs/changelog.md:4]: image

Comment (h4ck3rk3y) [reply docs/changelog.md:4]: no dependence!

Comment (leoporoli) [reply docs/changelog.md:4]: oh! i see, my mistake, go ahead!

**Review by leoporoli — APPROVED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (h4ck3rk3y) [reply docs/changelog.md:4]: no worries! thanks for being thorough

**Pull request merged**